azerothcore/azerothcore-wotlk
# fix(Core/Pets): Player summon's will not trigger unnecessary combat

A branch of code was using `IsPet()` where I think the author meant `IsSummon()`. Anyhow this created the issue linked below. 

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2838


## TESTS PERFORMED:
- Tested locally, fixed the issue. Nothing appears to be broken.

## HOW TO TEST THE CHANGES:

- Follow issue instructions and just see if the problems persists. 
- Should probably test other pet related stuff, see if nothing is broken etc.

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
